types/splitio.d.ts

```diff
@@ -1417,6 +1417,74 @@ declare namespace SplitIO {
      * @returns {TreatmentsWithConfig} The map with all the TreatmentWithConfig objects
      */
     getTreatmentsWithConfig(featureFlagNames: string[], attributes?: Attributes): TreatmentsWithConfig,
+    /**
+     * Returns a Treatments value, which is an object map with the treatments for the feature flags related to the given flagSet.
```

Comment:
`flagSet` -> `flag set`
Check other occurrences if the term `flagSet` or `flagSets` in descriptive comments